src/components/presentational/Error.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
Love that you added the extra touch of error handling, I appreciate the focus on user experience. Slowing down the call for the loading mask is also smart for this sort of app. In which cases might you keep things snappy vs slower?

Reply:
Great! One use case of a snappy response is error handling -- we could probably cut that down to 100 ms instead of 500ms. 

A slower response would be necessary when handling a large response from the API. E.g. if we are filtering or sorting thousands of arrays. 

Another time it's beneficial to have a slightly slower response is handling auth. IMO it's better UX as it gives the user a second or so to enter the app.